rclcpp/include/rclcpp/executor.hpp

```diff
@@ -263,6 +263,15 @@ class Executor
   void
   set_memory_strategy(memory_strategy::MemoryStrategy::SharedPtr memory_strategy);
 
+  /**
+   * Returns wake_after_execute_ flag
+   *
+   * \return wake_after_execute_ flag
+   */
```

Comment:
This doc block should have a brief above the block which starts with `///`, but also this doc block should be more descriptive. Like maybe "Return true if this executor should cause the executor to wake up after execution." or something like that.